app/services/metabase_embed_service.rb

```diff
@@ -0,0 +1,32 @@
+require 'jwt'
+
+class MetabaseEmbedService
+  METABASE_SITE_URL = 'https://metabase.entreprise.api.gouv.fr'.freeze
+
+  def initialize(question_id, params = {})
```

Comment:
Je me demande si ça serait pas plus clair d'avoir des named params:

```suggestion
  def initialize(question_id:, params: {})
```

Mais c'est que du style je te laisse choisir

Reply:
Dans ce cas là je trouve que c'est pas ouf niveau interface.

Après, honnêtement, c'est copilot qui a écrit ce code 😅